src/test/java/com/teragrep/pth10/utils.java

```diff
@@ -88,7 +91,7 @@ public void syntaxError(
                     RecognitionException e
             ) {
                 throw new IllegalStateException(
-                        "failed to parse at line " + line + ":" + charPositionInLine + " due to " + msg,
+                        "Assertions.failed to parse at line " + line + ":" + charPositionInLine + " due to " + msg,
```

Comment:
Exception message changed here as well.

Reply:
Pushed a fix for all accidental failed -> Assertions.failed conversions